style/js/scripts.js

```diff
@@ -303,22 +303,33 @@ $(document).ready(function() {
 
             if (!$(this).hasClass("active")) $(this).children('a').children('div').fadeOut(200);
         });
-    });	
+    });
+
+/* Fancybox */
+$(document).ready(function() {
+        $('.fancybox').mouseenter(function(e) {
+
+            $(this).children('a').children('div').fadeIn(300);
+        }).mouseleave(function(e) {
+
+            if (!$(this).hasClass("active")) $(this).children('a').children('div').fadeOut(200);
+        });
+    });
```

Comment:
You can remove this whole `IMAGE HOVER` section and replace it with CSS3 transitions. The portfolio scripts don't depend on JS transitions, you can use the same code you're currently using for `.defaultThumbnail`.